rotkehlchen/tests/fixtures/rotkehlchen.py

```diff
@@ -181,13 +181,20 @@ def augmented_unlock(user, password, create_new, initial_settings):
         _use_prepared_db(user_data_dir, use_custom_database)
         create_new = False
 
+    no_sanity_check = patch(
+        'rotkehlchen.db.dbhandler.db_schema_sanity_check',
+        new=lambda **_: None,
+    )
+
```

Comment:
I see this piece of code still repeating in multiple places. It's almost an one-liner so I get why one would hesitate to abstract it, but it would make easier to modify the tests in the future and this won't affect speed of tests visibly.

Edit: You have already made the functon but just not used it?

`mock_db_schema_sanity_check()`?

Reply:
> Edit: You have already made the functon but just not used it?

Yes, my fault :( Fixed now.